x11-wm/pekwm/pekwm-0.2.1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib desktop
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+S="${WORKDIR}"/${PN}\-${PV}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="contrib debug +jpeg +png themes truetype +xft xinerama +xpm"
+
+RDEPEND="
+	png? ( media-libs/libpng:0 )
+	jpeg? ( virtual/jpeg:0 )
+	xpm? ( x11-libs/libXpm )
+	x11-libs/libXrender
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	themes? ( dev-vcs/git )"
```

Comment:
This seems a bit weird. Also this USE flag has really no other function in the ebuild except a postinst message. I see there's "pekwm-themes" in the Manifest file, so maybe you can always or conditionally download the theme package and always or conditionally install the themes via this ebuild? 
https://gitweb.gentoo.org/repo/gentoo.git/tree/games-arcade/lbreakout2/lbreakout2-2.6.5-r2.ebuild I guess here's one example of how to do it, but by default src_unpack should also deal with it.

Reply:
The themes doesn't need to be downloaded anymore, there's a new tool called pekwm_theme, which can handle (select, download and install) this task. pekwm_theme needs git to run. I wasn't sure how to cope with this. I don't and didn't use "theme" feature at all, so I'm not sure what users expect from ebuild...

Reply:
I've been thinking of it during weekend and I can't find optimal solution. If one likes to use pekwm_theme then git is required. However, themes can be installed without using pekwm_theme, so you don't need git. If you don't have git and try to use pekwm_theme, pekwm_theme prints warning that git is required and exits. What's recommended? Put git as mandatory requirement or not? Or something else?

Reply:
Sounds like a good candidate for *optfeature*.
https://devmanual.gentoo.org/eclass-reference/optfeature.eclass/index.html

Reply:
Good point, it's exactly what we need. Commited (including a slight change to elog messages) eb4f7ad.